app/src/main/java/de/danoeh/antennapod/preferences/PreferenceController.java

```diff
@@ -825,15 +826,19 @@ private void buildEpisodeCleanupPreference() {
                 R.array.episode_cleanup_values);
         String[] entries = new String[values.length];
         for (int x = 0; x < values.length; x++) {
-            int v = Integer.parseInt(values[x]);
+            float v = Float.parseFloat(values[x]);
             if (v == UserPreferences.EPISODE_CLEANUP_QUEUE) {
```

Comment:
This probably needs to be adjusted
Changing the int to a float could break the following comparisons. See https://stackoverflow.com/questions/1088216/whats-wrong-with-using-to-compare-floats-in-java

I'd rather keep the values as ints. We could migrate the current settings (days to hours) in `UpdateManager`

Reply:
@mfietz Sorry for the long delay - I understand your concern of using float, though the risk here is almost minimal, given the possible values are ultimately all hardcoded in `arrays.xml` (with no float arithmetic involved).

I agree that keeping `int`s and migrating the settings from days to hours would be cleaner. If so, I believe the PR will 
- change `android:versionCode` from `1070195` to `1070196`
- perform the settings migration in `UpdateManager` when `oldVersionCode < 1070196`